package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-teachable-machine",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Version 1.3.0 since this adds a new functionality that it is backwards compatible
```suggestion
  "version": "1.3.0",
```